ci/test.sh

```diff
@@ -27,7 +28,8 @@ rm "${TOOLS_TEST_RESULTS_FILES}" \
 
 markEndOfBlock "Setup variables"
 
-cleanUnity
+cleanUnity "$(pwd)/workers/unity"
```

Comment:
may be nice to have pwd/workers/unity and pwd/test-project to be vars instead of inline

Reply:
can also be reused for the dotnet run args and `pushd`